includes/MediaWiki/Specials/SpecialWikibaseReconcileEdit.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+namespace MediaWiki\Extension\WikibaseReconcileEdit\MediaWiki\Specials;
+
+use HTMLForm;
+use MediaWiki\Extension\WikibaseReconcileEdit\MediaWiki\Request\EditRequestParser;
+use MediaWiki\Extension\WikibaseReconcileEdit\MediaWiki\Request\EditRequestSaver;
+use MediaWiki\Extension\WikibaseReconcileEdit\MediaWiki\WikibaseReconcileEditServices;
+use MediaWiki\Rest\LocalizedHttpException;
+use SpecialPage;
+use Status;
+use Wikibase\DataModel\Entity\PropertyId;
+use Wikibase\Lib\Store\EntityRevision;
+use Wikibase\Repo\WikibaseRepo;
+
+/**
+ * @license GPL-2.0-or-later
+ */
+class SpecialWikibaseReconcileEdit extends SpecialPage {
+
+	/** @var string */
+	private $reconcileDefaultJSON;
+
+	/** @var string */
+	private	$entityDefaultJSON;
+
+	/** @var EditRequestParser */
+	private $editRequestParser;
+
+	/** @var EditRequestSaver */
+	private $editRequestSaver;
+
+	public function __construct() {
+		parent::__construct( 'WikibaseReconcileEdit' );
+
+		$this->editRequestParser = WikibaseReconcileEditServices::getEditRequestParser();
+		$this->editRequestSaver = WikibaseReconcileEditServices::getEditRequestSaver();
```

Comment:
These two can be injected by specifying their service names in `extension.json`.

Reply:
good idea